addons/binding/org.openhab.binding.blebox/ESH-INF/config/config-description.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+    <config-description uri="blebox:config">
+
+        <parameter name="ip" type="text">
+                <label>IP Address</label>
+                <description>IP address of device</description>
+                <default>192.168.1.X</default>
+            </parameter>
+            
+            <parameter name="serialNumber" type="text">
```

Comment:
The parameter `serialNumber` doesn't seem to be used in code. Why have this parameter?